code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/Message.kt

```diff
@@ -114,6 +118,17 @@ internal fun Message(
              */
 
             val dialogWindow = (LocalView.current.parent as? DialogWindowProvider)?.window
+            dialogWindow?.let {
+                it.setLayout(
+                    ViewGroup.LayoutParams.MATCH_PARENT,
+                    ViewGroup.LayoutParams.MATCH_PARENT
+                )
+                if (Build.VERSION.SDK_INT >= 35) {
```

Comment:
The parent of this dialog is an unbounded compose view (because the size is set within `Message.kt`), does this still work for a non-full screen in-app message?

Reply:
It does work with non-full screen messages too, because when UI takeover is enabled we draw the web view over the entire screen size, irrespective of what the actual message size is. Here are some screenshots

Device running Android 16
![Screenshot_20250609_114111](https://github.com/user-attachments/assets/51fe87fd-d4f3-4584-a3e7-06c48a3f0fdb)
![Screenshot_20250609_114204](https://github.com/user-attachments/assets/141648f6-aa8e-4e35-acb3-b69c940e08c1)

Device running Android 30
![Screenshot_20250609_114410](https://github.com/user-attachments/assets/fa7a9f68-8abb-4dca-969b-6ce1946143ee)
![Screenshot_20250609_115003](https://github.com/user-attachments/assets/ba8eb28d-fea6-4175-b9a2-17937a037661)